gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -715,3 +847,56 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class WriteToVcf(PTransform):
+  """A PTransform for writing to VCF files."""
+
+  def __init__(self,
+               file_path_prefix,
+               file_name_suffix='',
+               num_shards=0,
```

Comment:
I think this should be set to 1 by default. People generally expect one VCF file (unless you have a very large output that you like to shard it and join it yourself; but that's likely not common as people would filter them prior to exporting). We have to also see how long it would take to output a ~5GB file.

Reply:
Done.